src/portable/renesas/rusb2/rusb2_type.h

```diff
@@ -48,6 +48,7 @@ extern "C" {
 /* Start of definition of packed structs (used by the CCRX toolchain) */
 TU_ATTR_PACKED_BEGIN
 TU_ATTR_BIT_FIELD_ORDER_BEGIN
+#pragma pack(2)
```

Comment:
I found that using the armclang compiler(ac6) without adding this code will cause it to not run properly.

Reply:
If pack is wrong it should be fixed in `src/common/tusb_compiler.h`